weed/storage/volume.go

```diff
@@ -23,6 +28,9 @@ type Volume struct {
 
 	dataFileAccessLock sync.Mutex
 	lastModifiedTime   uint64 //unix time in seconds
+
+	lastCompactingIndexOffset       uint64
+	incrementedHasUpdatedIndexEntry map[uint64]keyField
```

Comment:
I am only talking about incrementedHasUpdatedIndexEntry. The map only used locally in makeupDiff()